Source/santactl/Commands/SNTCommandRule.m

```diff
@@ -225,18 +226,16 @@ - (void)runWithArguments:(NSArray *)arguments {
   }
 
   if (!importRules && cleanupType != SNTRuleCleanupNone) {
-    switch (cleanupType) {
-      case SNTRuleCleanupNonTransitive:
-        [self printErrorUsageAndExit:@"--clean can only be used with --import"];
-        break;
-      case SNTRuleCleanupAll:
-        [self printErrorUsageAndExit:@"--clean-all can only be used with --import"];
-        break;
-      default:
-        // This is a programming error.
-        LOGE(@"Unexpected SNTRuleCleanupType %ld", cleanupType);
-        exit(EXIT_FAILURE);
-    }
+    [[self.daemonConn remoteObjectProxy]
+        databaseRuleAddRules:@[]
+                 ruleCleanup:cleanupType
```

Comment:
With 0 rules and a `cleanupType != SNTRuleCleanupNone` (from if statement) - this will be a no-op when hitting the new check in `addRules:ruleCleanup:error:`

Reply:
I'd inverted this and forgot to put it back. Fixed